src/OrleansRuntime/MultiClusterNetwork/IGossipChannel.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Threading.Tasks;
+using Orleans.Runtime.Configuration;
+
+
+namespace Orleans.Runtime.MultiClusterNetwork
+{
+    /// <summary>
+    /// Interface for gossip channel.
+    /// 
+    /// A gossip channel stores multicluster data (configuration, gateways) and exchanges
+    /// this data with silos using a gossip-style communication, offering
+    /// two different methods (push or push-and-pull).
+    /// </summary>
+    public interface IGossipChannel
+    {
+        /// <summary>
+        /// Initialize the channel with given configuration.
+        /// </summary>
+        /// <param name="config"></param>
+        /// <returns></returns>
+        Task Initialize(GlobalConfiguration globalconfig, string connectionstring);
```

Comment:
Do we need to pass all of `GlobalConfiguration` here? `AzureTableBasedGossipChannel` appears to be using only `GlobalServiceId` property of it. I think it is better to provide the minimally needed data, and expand if necessary later then to share everything.